pyproject.toml

```diff
@@ -30,15 +21,35 @@ classifiers = [
     "Programming Language :: Python :: Implementation :: CPython",
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
+dynamic = ["dependencies"]
+readme = "README.md"
+authors = [{name = "NI", email = "opensource@ni.com"}]
+maintainers = [{name = "NI", email = "opensource@ni.com"}]
+requires-python = '>=3.9,<4.0'
+
+[project.urls]
+repository = "https://github.com/ni/hightime"
+documentation = "https://hightime.readthedocs.io"
+
+[build-system]
+requires = ["poetry-core>=2.1,<3.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+requires-poetry = '>=2.1,<3.0'
 
 [tool.poetry.dependencies]
```

Comment:
the diff makes it look like we're keeping this empty table, is this intentional?

Reply:
https://github.com/python-poetry/poetry/issues/10557